docs/appendices/release-notes/5.9.1.rst

```diff
@@ -47,6 +47,15 @@ See the :ref:`version_5.9.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed behavior of comparing (``=``, ``>``, etc. ) a column of type
+  :ref:`CHAR<type-char>` with string literals, so that any trailing whitespaces
+  are ignored for both sides of the comparison.
```

Comment:
Shouldn't we mention that it's for PG compatibility?

Reply:
can do, thx